icicle/include/fields/field.cuh

```diff
@@ -124,6 +124,17 @@ public:
    */
   static constexpr HOST_DEVICE_INLINE unsigned num_of_reductions() { return CONFIG::num_of_reductions; }
 
+  static __host__ __device__ unsigned num_bits(const Field& x)
+  {
+    unsigned ret = 32 * TLC;
```

Comment:
Please use sizeof(x.limbs[0]) rather than hardcoding 32, in case we modify the limb size in the future.
Also please add a short comment explaining what it does.

Reply:
done